WangscapeGui/optionseditor.cpp

```diff
@@ -0,0 +1,64 @@
+#include "optionseditor.h"
```

Comment:
Is all-lowercase for window classes a Qt convention, or can we rename these to `OptionsEditor.h`, `MainWindow.cpp`, etc.?

Reply:
They were generated by QtCreator, I'll change them.

Reply:
Auto-generated headers file, `ui_*`, will be still all lower-case.

Reply:
Ops, wrong. They were just some cached version, fixed to `ui_MainWindow.h` and `ui_OptionsEditor.h`.